packages/testlab/package.json

```diff
@@ -4,10 +4,12 @@
   "description": "A collection of test utilities we use to write LoopBack tests.",
   "main": "index",
   "scripts": {
-    "build": "tsc && npm run build:es2015",
-    "build:es2015": "tsc --target es2015 --outDir lib-es2015",
+    "build": "npm run build:es2015 && npm run build:es2017",
+    "build:es2015": "tsc --target es2015 --outDir lib6",
+    "build:es2017": "tsc --target es2017 --outDir lib",
```

Comment:
You mean `--target`, right? See my https://github.com/strongloop/loopback-next/pull/236#discussion_r113874485, `es7` does not exist.